submissions/emmanuelEsuong/css/styles-mediaQuery.css

```diff
@@ -0,0 +1,264 @@
+/* media-query (tablet)*/
+@media screen and (min-width: 765px) and (max-width: 900px) {
+        /* signUp page */
+    /* formfield */
+    #s-form {
+        border-radius: 25px;
+        border: solid 1px;
```

Comment:
Why do you have 2 border attribute declaration. Make it one.

Reply:
Updated